app/controllers/concerns/jobseekers/wizardable.rb

```diff
@@ -7,6 +7,10 @@ def professional_status_fields
     %i[qualified_teacher_status qualified_teacher_status_year qualified_teacher_status_details statutory_induction_complete]
   end
 
+  def qualifications_fields
+    []
```

Comment:
is this a part 1 thing that will have the array filled up in subsequent parts?

Reply:
No - it's like the reference_fields later in this file. These steps don't have any fields, but they need this method in order to be step-shaped for the wizard.